### PR TITLE
WSTEAMA-266 Homepage SEO attributes

### PR DESCRIPTION
**Pull request by LilyL0u**
Resolves JIRA number 266

Overall changes
======

Added the LinkedData component onto the new home page, in the structure of the schema provided by @tpchandler 

Testing
======

Unit test included. Tested also on [www.schema.org](www.schema.org)

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -24,12 +25,39 @@ interface HomePageProps {
...
+      })),
+    )
+    .flat()
+    .map((listItem, index) => {
+      return {
+        ...listItem,
+        position: index + 1,
+      };
+    });
+  const itemList = {
```
Comment (karinathomasbbc) [inline src/app/pages/HomePage/HomePage.tsx]: Let's add a whitespace here 
```suggestion
    });

  const itemList = {
```

File: src/app/pages/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -24,12 +25,39 @@ interface HomePageProps {
...
+    itemListElement,
+    '@type': 'ItemList',
+    name: brandName,
+    numberOfItems: itemListElement.length,
+  };
+  console.log(JSON.stringify(itemListElement, null, 2));
```
Comment (karinathomasbbc) [inline src/app/pages/HomePage/HomePage.tsx]: Let's remove this 😄 
```suggestion
```

File: src/app/pages/HomePage/index.test.tsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { data as kyrgyzHomePageData } from '#data/kyrgyz/homePage/index.json';
 import { Helmet } from 'react-helmet';
 import { render } from '../../components/react-testing-library-with-providers';
 import HomePage from './HomePage';
+import LinkedData from '../../components/LinkedData';
```
Comment (karinathomasbbc) [inline src/app/pages/HomePage/index.test.tsx]: I think this can now be deleted (is also causing `yarn build` to fail)

```suggestion
```

**Review by karinathomasbbc — APPROVED** [review]
Yay! 🎉 🎸

**Review by shayneahchoon — APPROVED** [review]

**Review by amoore108 — APPROVED** [review]
Nice!

**Review by karinathomasbbc — APPROVED** [review]
🌮

**Pull request merged**